index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav>
+        <!-- navigation bar begins here -->
+            <a href="#"><h2 class="logo">Diana</h2></a>
+            <div class="maimenu">
+                <a href=""><img src="image/Union.svg" alt="union-icon"></a>
+            </div>
+    </nav>
+</header>
```

Comment:
Kindly move the logo to the left and and the menu bar to the right a little bit so you can match the given design better 😄 👍 
![image](https://user-images.githubusercontent.com/87448628/185610466-cd248346-2970-41f2-bd1b-2270ac0f5318.png)